### PR TITLE
rgw/pubsub: clarify pubsub zone configuration

### PR DESCRIPTION
**Pull request by yuvalif**
Signed-off-by: Yuval Lifshitz <ylifshit@redhat.com>

more detailed documentation on how to configure a pubsub zone.

### CODE REVIEW

**Comment by yuvalif** [comment]
@ofriedma please have a look. let me know if more clarifications are needed?

File: doc/radosgw/pubsub-module.rst [hunk 1]
```diff
@@ -68,28 +84,18 @@ The oid prefix for the stored events.
...
+The tier configuration could be set using the following command:
 
 ::
 
    # radosgw-admin zone modify --rgw-zonegroup={zone-group-name} \
                                 --rgw-zone={zone-name} \
                                 --tier-config={key}={val}[,{key}={val}]
 
-The ``key`` in the configuration specifies the configuration variable that needs to be updated, and
+Where the ``key`` in the configuration specifies the configuration variable that needs to be updated, and
```
Comment (ofriedma) [inline doc/radosgw/pubsub-module.rst]: I think we should be more clear about what are the keys necessary for the pubsub to work

Comment (cbodley) [reply doc/radosgw/pubsub-module.rst]: aren't these spelled out in `PubSub Zone Configuration Parameters` section directly above?

File: doc/radosgw/pubsub-module.rst [hunk 2]
```diff
@@ -35,9 +35,25 @@ be accessed directly, but need to be pulled and acked using the new REST API.
...
 
-PubSub Tier Type Configuration
--------------------------------
+PubSub Zone Configuration
+-------------------------
+
+The pubsub sync module requires the creation of a new zone in a `Multisite`_ environment.
+First, a master zone must exist (see: :ref:`master-zone-label`), 
+then a secondary zone should be created (see :ref:`secondary-zone-label`).
+None that, When doing the actual creation of the secondary zone zone, its tier type must be set to ``pubsub``:
```
Comment (cbodley) [inline doc/radosgw/pubsub-module.rst]: When->when

secondary zone ~~zone~~

File: doc/radosgw/pubsub-module.rst [hunk 3]
```diff
@@ -325,7 +334,7 @@ Request parameters:
 
 - topic-name: name of topic
 
-Note that when the bucket is deleted, any notification defined on it is also deleted
+.. note:: when the bucket is deleted, any notification defined on it is also deleted
```
Comment (cbodley) [inline doc/radosgw/pubsub-module.rst]: when->When

**Comment by cbodley** [comment]
jenkins render docs

File: doc/radosgw/pubsub-module.rst (hunk 2 above)
Comment (yuvalif) [reply doc/radosgw/pubsub-module.rst]: > aren't these spelled out in PubSub Zone Configuration Parameters section directly above?
i think that this is an alternative to setting the zone parameters in a conf file. these CLI commands are showing how to set the parameters via CLI - will change the headline so it is clear

File: doc/radosgw/pubsub-module.rst (hunk 1 above)
Comment (yuvalif) [reply doc/radosgw/pubsub-module.rst]: i think that this is an alternative to setting the zone parameters in a conf file. these CLI commands are showing how to set the parameters via CLI - will change the headline so it is clear

**Comment by yuvalif** [comment]
jenkins render docs

**Pull request merged**